src/components/Config.js

```diff
@@ -220,7 +220,7 @@ function ConfigImpl({
           <ToggleSwitch
             options={optionsRule}
             name="mode"
-            value={configState.mode && configState.mode.toLowerCase()}
+            value={configState.mode}
             onChange={handleInputOnChange}
```

Comment:
There is a breaking change clash on it's `dev` branch

https://github.com/Dreamacro/clash/blob/60bad66bc30b04b8c05d739a21773f07e0f10ba9/tunnel/mode.go#L60-L71